seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/oceanbase/OceanBaseMySqlTypeConverter.java

```diff
@@ -510,4 +554,142 @@ public BasicTypeDefine<OceanBaseMysqlType> reconvert(Column column) {
 
         return builder.build();
     }
+
+    public static List<Column> convert(DatabaseMetaData metadata, TablePath tablePath)
```

Comment:
Can we merge this into https://github.com/apache/seatunnel/blob/e4e12245b90e994bce27ea90ac044e722888a8ae/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/oceanbase/OceanBaseMySqlTypeConverter.java#L109 ?

Reply:
Yes, I've incorporated this method